ipatests/test_ipapython/test_ipautil.py

```diff
@@ -55,8 +55,10 @@ def test_ip_address():
         ('241.1.2.3',),
         ('169.254.1.2',),
         ('10.11.12.0/24',   (10, 11, 12, 0),   24),
+        ('10.0.0.255',      (10, 0, 0, 255),   24),
```

Comment:
Test fails with:
    def check_ipaddress():
        try:
            ip = ipautil.CheckedIPAddress(addr, match_local=False)
            assert ip.words == words and ip.prefixlen == prefixlen
        except Exception:
>           assert words is None and prefixlen is None
E           assert ((10, 0, 0, 255) is None)

CheckedIPAddress(10.0.0.255).prefixlen is 8 not 24